internal/server/wrap_handler.go

```diff
@@ -38,12 +38,34 @@ type ErrorVars struct {
 	EnvironmentVars
 }
 
+type ExpandedError interface {
+	Title() string
+	Data() interface{}
+}
+
+func LoggerRequest(l *slog.Logger, r *http.Request, err error) {
```

Comment:
Not sure how necessary this is now. The middleware will already log out every request, so this may be doubling up. Start up the app locally and have a look in the logs to see if it is still needed.

Reply:
It seems to be necessary from what I've found, I tried running it with these LoggerRequests commented out and it seemingly didn't log any of the requests I was getting with it uncommented